elki/src/main/java/de/lmu/ifi/dbs/elki/math/statistics/dependence/MCDEDependenceMeasure.java

```diff
@@ -0,0 +1,173 @@
+package de.lmu.ifi.dbs.elki.math.statistics.dependence;
```

Comment:
Can you please add the usual ELKI license (AGPL-3), so this code can be merged?

Reply:
In the current master branch, we have simplified the package hierarchy from `de.lmu.ifi.dbs.elki` to just `elki`. It would be best if you can rebase your pull request on the current master branch.